.github/workflows/dependecyUpdates.yml

```
@@ -41,7 +41,7 @@ jobs:
 
       # Run dependencyUpdates task
       - name: Run dependencyUpdates task
-        uses: gradle/gradle-build-action@937999e9cc2425eddc7fd62d1053baf041147db7
+        uses: gradle/gradle-build-action@0d13054264b0bb894ded474f08ebb30921341cee
```

Use version instead of commit id
```suggestion
        uses: gradle/gradle-build-action@v2.1.4
```